easybuild/easyconfigs/o/occt/occt-7.8.0-foss-2023a.eb

```diff
@@ -0,0 +1,47 @@
+easyblock = 'CMakeMake'
+
+name = 'occt'
+version = '7.8.0'
+
+homepage = 'https://www.opencascade.com/'
+description = """Open CASCADE Technology (OCCT) is an object-oriented C++
+class library designed for rapid production of sophisticated domain-specific
+CAD/CAM/CAE applications."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
```

Comment:
I see no indications that this uses either BLAS or MPI, we can definitely move this. In fact, i'm pretty sure it's mostly just header only so I say lets try moving it to GCCcore.

Reply:
You are right. The toolchain is changed to GCCcore in my last commit

Reply:
your PR still contains the old version, clear it out with `--update-pr 19864 :occt-7.8.0-foss-2023a.eb`